packages/generator/README.md

```diff
@@ -7,6 +7,9 @@
 Measurement Plug-In Code Generator for Python (`ni-measurement-plugin-sdk-generator`) is a
 tool for generating reusable measurement plug-ins using gRPC services.
 
+Measurement Plug-In Client Generator for Python (`ni-measurement-plugin-client-generator`) is a
+tool for generating python clients to interactive with the measurement plug-ins.
+
 For installation and usage, see [Measurement Plug-In SDK for Python (`ni-measurement-plugin-sdk-service`)](https://pypi.org/project/ni-measurement-plugin-sdk-service/).
```

Comment:
This link text is wrong because -sdk and -sdk-service are separate packages now.

Reply:
I observed -sdk-service is referred as -sdk in other README.md files like 
[https://github.com/ni/measurement-plugin-python/tree/main/packages/sdk#introduction](url).
[https://github.com/ni/measurement-plugin-python/blob/main/packages/service/README.md#measurement-plug-in-sdk-for-python](url)
Should we consider updating them too?

Reply:
Yes, we should fix that. It's confusing to call multiple packages the same thing.

Reply:
I have updated the relevant sections to install the -sdk package directly in all README files, instead of separately installing the -sdk-service and -sdk-generator packages.